tests/app/api/services/test_abr.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+
+from app.api.services import abr_service
+from app.api.business.errors import AbrError
+import requests
+import mock
+from mock import patch
+
+
+class TestAbrService():
+
+    def mocked_find_business_by_abn(self):
+        data = '<ABR><response><stateCode>NSW</stateCode><postcode>2750</postcode>'\
+            '<organisationName>yay</organisationName></response></ABR>'
+        return data
+
+    def mocked_payload_exception(self):
+        data = '<ABR><response><exception><exceptionDescription>Search text is not a '\
+            'valid ABN or ACN</exceptionDescription><exceptionCode>WEBSERVICES</exceptionCode>'\
+            '</exception></response></ABR>'
+        return data
+
+    @mock.patch("app.api.services.abr_service.call_abr_api")
+    def test_parsed_data(self, mocked_find_business_by_abn):
+        """ test expected parsed data"""
+        expected_parsed_data = {'state': 'NSW', 'organisation_name': 'yay', 'postcode': '2750'}
+        data = abr_service.get_data(self.mocked_find_business_by_abn())
+        assert data == expected_parsed_data
+
+    @mock.patch("app.api.services.abr_service.call_abr_api")
+    def test_payload_exceptions(self, mocked_payload_exception):
+        """ test payload exception"""
+        expected_msg = 'WEBSERVICES: Search text is not a valid ABN or ACN'
+        result = abr_service.get_abr_exception(self.mocked_payload_exception())
```

Comment:
Should we test these scenarios?

1. When a code exists but no description
2. When a description exists but no code
3. When no exception is in the response